libs/ha/ha_comp_libs.py

```diff
@@ -191,19 +192,30 @@ def get_ha_log_prop(node_obj, log_name: str, kvalue: int, fault_tolerance: bool
             output = log_list
         resp_dict = {'source': [], 'resource_status': [], 'resource_type': [], 'generation_id': [],
                      'node_id': [], 'resource_id': []}
+        return HAK8SCompLib.get_properties_data(output, resp_dict)
```

Comment:
do we want to do these optimizations without testing if they are good?

Reply:
> do we want to do these optimizations without testing if they are good?

I just have used pycharm techniq, which will generate static function from the section which we select. (refector ->extract method) May be @seagate-sarang-sawant can suggest more here as he is suggested the same in one PR to reduce complexity of the function. Please suggest.

Reply:
As @Pradeep-B-Thangaraj checked and approved, I'm ok with this.